File: 2. python/1. basics/lecture 4/count digits.py
class solution:
     def __init__(self, n):
         count = 0
         while n != 0:
             n = n // 10
             count += 1
         print("Number of digits:", count)

if __name__ == "__main__":
    n = int(input("Enter number:\n"))
    print("output:")
    pattern = solution(n)

File: 1. cpp/1. basics/lecture 4/count digits.cpp

#include <iostream>
using namespace std;
class solution {
public:
    solution(int n) {
        int count = 0;
        while (n != 0) {
            n = n / 10;
            count++;
        }
        cout << "Number of digits: " << count << endl;
    }
};

/* output:



*/

int main() {
  int n;
  cout<<"Enter number:"<<endl;
  cin>>n;

  cout<<"output:"<<endl;
  solution pattern(n);

}